gcp/README.md

```diff
@@ -0,0 +1,59 @@
+# Setup your secrets in GCP
+
+In this setup we integrate the secrets exercise with GCP GKE and let pods consume secrets from the GCP Secret manager. If you want to know more about integrating secrets with GKE, check [this link](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp).
```

Comment:
please note the experimental nature, so we can merge

Reply:
Added 'Experimental' but didn't add the issues - can we do without the links?

Reply:
sure ;)